public/pages/CreateMonitor/containers/DefineMonitor/utils/mappings.js

```diff
@@ -43,6 +43,8 @@ export function getFieldsFromProperties(properties, dataTypes, path) {
 }
 
 export function getTypeFromMappings(mappings, dataTypes, path = '') {
+  // properties: { "field_name":{"properties": ...} }
+  // properties: { "field_name":{"type": "text"} }
```

Comment:
Can remove this commented code

Reply:
Bowen left it here to let people understand the structure of index mapping better: https://github.com/opensearch-project/alerting-dashboards-plugin/pull/44#discussion_r692454667

Reply:
Ah, could we add a line above this comment then labeling it, something like:
```
// Possible structure of index mappings:
```

Reply:
Added